Can't build executable broker on Mac OS

**Description**

I packaged projects successfully on Mac OS, in the process of running, get the following error:

```
java.lang.UnsatisfiedLinkError: failed to load the required native library
	at io.netty.channel.epoll.Epoll.ensureAvailability(Epoll.java:80) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.netty.channel.epoll.EpollEventLoop.<clinit>(EpollEventLoop.java:51) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.netty.channel.epoll.EpollEventLoopGroup.newChild(EpollEventLoopGroup.java:150) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.netty.channel.epoll.EpollEventLoopGroup.newChild(EpollEventLoopGroup.java:35) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:84) ~[netty-common-4.1.58.Final.jar:4.1.58.Final]
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:58) ~[netty-common-4.1.58.Final.jar:4.1.58.Final]
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:47) ~[netty-common-4.1.58.Final.jar:4.1.58.Final]
	at io.netty.channel.MultithreadEventLoopGroup.<init>(MultithreadEventLoopGroup.java:59) ~[netty-transport-4.1.58.Final.jar:4.1.58.Final]
	at io.netty.channel.epoll.EpollEventLoopGroup.<init>(EpollEventLoopGroup.java:112) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.netty.channel.epoll.EpollEventLoopGroup.<init>(EpollEventLoopGroup.java:99) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.netty.channel.epoll.EpollEventLoopGroup.<init>(EpollEventLoopGroup.java:76) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.atomix.cluster.messaging.impl.NettyMessagingService.initEventLoopGroup(NettyMessagingService.java:317) ~[atomix-cluster-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.atomix.cluster.messaging.impl.NettyMessagingService.start(NettyMessagingService.java:244) ~[atomix-cluster-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.Broker.commandApiTransportStep(Broker.java:284) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.Broker.lambda$initStart$1(Broker.java:209) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.bootstrap.StartProcess.lambda$startStepByStep$2(StartProcess.java:63) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:92) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.bootstrap.StartProcess.startStepByStep(StartProcess.java:61) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:92) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.bootstrap.StartProcess.start(StartProcess.java:46) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.Broker.internalStart(Broker.java:180) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.util.LogUtil.doWithMDC(LogUtil.java:21) ~[zeebe-util-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.Broker.start(Broker.java:160) ~[zeebe-broker-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at io.zeebe.broker.StandaloneBroker.run(StandaloneBroker.java:60) ~[zeebe-distribution-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:804) [spring-boot-2.4.2.jar:2.4.2]
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:788) [spring-boot-2.4.2.jar:2.4.2]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:333) [spring-boot-2.4.2.jar:2.4.2]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1311) [spring-boot-2.4.2.jar:2.4.2]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1300) [spring-boot-2.4.2.jar:2.4.2]
	at io.zeebe.broker.StandaloneBroker.main(StandaloneBroker.java:47) [zeebe-distribution-0.27.0-SNAPSHOT.jar:0.27.0-SNAPSHOT]
Caused by: java.lang.ExceptionInInitializerError
	at io.netty.channel.epoll.Epoll.<clinit>(Epoll.java:39) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	... 29 more
Caused by: java.lang.IllegalStateException: Only supported on Linux
	at io.netty.channel.epoll.Native.loadNativeLibrary(Native.java:244) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.netty.channel.epoll.Native.<clinit>(Native.java:69) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	at io.netty.channel.epoll.Epoll.<clinit>(Epoll.java:39) ~[netty-transport-native-epoll-4.1.58.Final-linux-x86_64.jar:4.1.58.Final]
	... 29 more
```

Related topic in the forum: https://forum.zeebe.io/t/source-code-package-success-run-failure/1984/2

--- a/atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java
+++ b/atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java
@@ -41,6 +41,7 @@ import io.netty.channel.EventLoopGroup;
 import io.netty.channel.ServerChannel;
 import io.netty.channel.SimpleChannelInboundHandler;
 import io.netty.channel.WriteBufferWaterMark;
+import io.netty.channel.epoll.Epoll;
 import io.netty.channel.epoll.EpollEventLoopGroup;
 import io.netty.channel.epoll.EpollServerSocketChannel;
 import io.netty.channel.epoll.EpollSocketChannel;
@@ -263,7 +264,7 @@ public class NettyMessagingService implements ManagedMessagingService {
       return CompletableFuture.completedFuture(this);
     }
 
-    initEventLoopGroup();
+    initTransport();
     return bootstrapServer()
         .thenRun(
             () -> {
@@ -332,22 +333,24 @@ public class NettyMessagingService implements ManagedMessagingService {
     return CompletableFuture.completedFuture(null);
   }
 
-  private void initEventLoopGroup() {
-    // try Epoll first and if that does work, use nio.
-    try {
-      clientGroup =
-          new EpollEventLoopGroup(0, namedThreads("netty-messaging-event-epoll-client-%d", log));
-      serverGroup =
-          new EpollEventLoopGroup(0, namedThreads("netty-messaging-event-epoll-server-%d", log));
-      serverChannelClass = EpollServerSocketChannel.class;
-      clientChannelClass = EpollSocketChannel.class;
-      return;
-    } catch (final Exception e) {
-      log.debug(
-          "Failed to initialize native (epoll) transport. " + "Reason: {}. Proceeding with nio.",
-          e.getMessage(),
-          e);
+  private void initTransport() {
+    if (Epoll.isAvailable()) {
+      initEpollTransport();
+    } else {
+      initNioTransport();
     }
+  }
+
+  private void initEpollTransport() {
+    clientGroup =
+        new EpollEventLoopGroup(0, namedThreads("netty-messaging-event-epoll-client-%d", log));
+    serverGroup =
+        new EpollEventLoopGroup(0, namedThreads("netty-messaging-event-epoll-server-%d", log));
+    serverChannelClass = EpollServerSocketChannel.class;
+    clientChannelClass = EpollSocketChannel.class;
+  }
+
+  private void initNioTransport() {
     clientGroup =
         new NioEventLoopGroup(0, namedThreads("netty-messaging-event-nio-client-%d", log));
     serverGroup =

